tooling/build/scripts/publishing/index.ts

```diff
@@ -1,6 +1,7 @@
 import * as fs from "fs"
 import * as path from "path"
 import { performance } from "perf_hooks"
+import { ISOMER_PAGE_LAYOUTS } from "@opengovsg/isomer-components"
```

Comment:
issue: if I'm not wrong the `tooling/build` realm is quite different from studio and the components package, so it would not be possible for us to directly import components-related stuff into the files here (we don't list the components package as a dependency). So we might have to hard code this part here.